include/wx/cmdargs.h

```diff
@@ -31,19 +31,17 @@ class WXDLLIMPEXP_BASE wxCmdLineArgsArray
     wxCmdLineArgsArray() { m_argsA = NULL; m_argsW = NULL; }
 
     template <typename T>
-    wxCmdLineArgsArray& operator=(T **argv)
```

Comment:
I don't think this class is supposed to be part of the public API, it's (intentionally) not documented and it seems pretty unlikely that anybody uses it in their code, so while we could keep this assignment operator, I don't think it's worth doing it. Does anybody disagree?